### PR TITLE
fix ru translation

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/public/ru/how-to/afterschool.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ We provide a variety of [fun, hour-long tutorials](<%= resolve_url('https://code
 **Покажите им вдохновляющее видео:**
 
   * Оригинальное начальное видео от Code.org, в котором вы сможете увидеть Билла Гейтса, Марка Цукерберга и звезду NBA Криса Боша ([1минутная версия](https://www.youtube.com/watch?v=qYZF6oIZtfc), [5-минутная версия](https://www.youtube.com/watch?v=nKIu9yen5nc) и [9-минутная версия](https://www.youtube.com/watch?v=dU1xS07N-FA))
-  * В[запуске видео Час кода 2013](https://www.youtube.com/watch?v=FC5FbmsH4fw) или <% if @country == 'uk' %>[Час кода 2015 видео](https://www.youtube.com/watch?v=7L97YMYqLHc)<% еще %>[Час кода 2015 видео](https://www.youtube.com/watch?v=7L97YMYqLHc) <% end %>
+  * В[запуске видео Час кода 2013](https://www.youtube.com/watch?v=FC5FbmsH4fw) или <% if @country == 'uk' %>[Час кода 2015 видео](https://www.youtube.com/watch?v=7L97YMYqLHc) <% else %> [Час кода 2015 видео](https://www.youtube.com/watch?v=7L97YMYqLHc) <% end %>
```
Comment (ashercodeorg) [inline pegasus/sites.v3/hourofcode.com/i18n/public/ru/how-to/afterschool.md:49]: Should there be spaces around the else brackets? If so, I'm curious why.

Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/i18n/public/ru/how-to/afterschool.md:49]: It doesn't require spaces, but I think it's easier to read.

Comment (ashercodeorg) [reply pegasus/sites.v3/hourofcode.com/i18n/public/ru/how-to/afterschool.md:49]: So the spaces outside the brackets get stripped? How does one indicate that starting or ending whitespace shouldn't be stripped?

Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/i18n/public/ru/how-to/afterschool.md:49]: Yeah. Even if I put more whitespace between `или` and the `if condition` it gets stripped. I'm not sure how you would keep the whitespace. When would you want to have extra whitespace?

**Comment by tanyaparker** [comment]
(Circle failure is Big Game - unrelated)

**Pull request merged**